library/src/main/java/com/mux/video/upload/internal/TranscoderContext.kt

```diff
@@ -178,8 +178,6 @@ internal class TranscoderContext private constructor(
                 inputAudioFormat = format;
                 extractor.selectTrack(i)
             }
-
-            uploadInfo = uploadInfo.update(shouldStandardize = shouldStandardize)
```

Comment:
question(blocking): wasn't "shouldStandardize" here a different concept, aka: even if the user _would like to standardize_ (generically), we've under the hood decided that _we shouldn't standardize for this particular resource_? (See: https://github.com/muxinc/android-upload-sdk/blob/releases/v0.4.1/library/src/main/java/com/mux/video/upload/internal/TranscoderContext.kt#L126-L170)

Reply:
On the one hand, it was. But the field wasn't actually used anywhere. Its purpose was duplicated elsewhere, so I just used it

Reply:
Fwiw the concept analogous to the one in linked to `TranscoderContext` logic is expressed as an "inspection result" on iOS to make that ^ distinction obvious.